README.md

```diff
@@ -15,50 +15,54 @@ npm install --save-dev @enact/jsdoc-to-ts
 Assuming this directory is a peer of the Enact source and you want to write this into an installed (from npm) version of enact:
 
 ```bash
-node -e "['core', 'ui', 'moonstone', 'i18n', 'webos', 'spotlight'].forEach(p => require('.')({
-  package: '../enact/packages/' + p,
-  output: require('fs').writeFileSync,
-  ignore: ['node_modules', 'ilib'],
-  importMap: {
-    ui: '@enact/ui',
-    moonstone: '@enact/moonstone',
-    core: '@enact/core',
-    webos: '@enact/webos',
-    spotlight: '@enact/spotlight',
-    i18n: '@enact/i18n'
-  },
-  outputPath: '<path to installation>/node_modules/@enact/' + p
-}))"
+node -e "import('./index.js').then(({default: jsdocToTs}) => {
+  ['core', 'ui', 'moonstone', 'i18n', 'webos', 'spotlight'].forEach(p => jsdocToTs({
+    package: '../enact/packages/' + p,
+    output: require('fs').writeFileSync,
+    ignore: ['node_modules', 'ilib'],
+    importMap: {
+      ui: '@enact/ui',
+      moonstone: '@enact/moonstone',
+      core: '@enact/core',
+      webos: '@enact/webos',
+      spotlight: '@enact/spotlight',
+      i18n: '@enact/i18n'
+    },
+    outputPath: '<path to installation>/node_modules/@enact/' + p
+  }))
+})"
 ```
 NOTE: Replace `<path to installation>` with the directory you wish to update.
 
 ## Working with Linked Enact
 
 The following command will parse the Enact source into a local `types` directory so you can use TypeScript with linked Enact:
 ```bash
-node -e "['core', 'ui', 'moonstone', 'i18n', 'webos', 'spotlight'].forEach(p => require('.')({
-  package: '<path to enact>/packages/' + p,
-  output: (filepath, content) => {
-    const path = require('path');
-    const dir = path.dirname(filepath);
-    const wfs = require('fs').writeFileSync;
-    // recursively create the path to the folder in types
-    require('mkdirp').sync(dir);
-    // write a package.json with a types member pointing to the .d.ts file
-    wfs(path.join(dir, 'package.json'), '{\"types\": \"' + path.basename(filepath) + '\"}');
-    // write the .d.ts file
-    wfs(filepath, content);
-  },
-  importMap: {
-    ui: '@enact/ui',
-    moonstone: '@enact/moonstone',
-    core: '@enact/core',
-    webos: '@enact/webos',
-    spotlight: '@enact/spotlight',
-    i18n: '@enact/i18n'
-  },
-  outputPath: '<path to installation>/types/@enact/' + p
-}))"
+node -e "import('./index.js').then(({default: jsdocToTs}) => {
+    ['core', 'ui', 'moonstone', 'i18n', 'webos', 'spotlight'].forEach(p => jsdocToTs('.')({
```

Comment:
I don't think this command works. Is this right?

Reply:
There's an error in the guide. I will fix it.